pytplot/Interactive2DPlot.py

```diff
@@ -0,0 +1,136 @@
+import pyqtgraph as pg
+import numpy as np
+import pytplot
+from pyqtgraph.Qt import QtGui, QtCore
+
+
+def get_data(names):
+    # Just grab variables that are spectrograms.
+    valid_vars = list()
+    for n in names:
+        if pytplot.data_quants[n].spec_bins is not None:
+            valid_vars.append(n)
+    return valid_vars
+
+
+def get_plot_labels(names):
+    # Get labels and axis types for plots.
+    plot_labels = {}
+    for n in names:
+        if pytplot.data_quants[n].spec_bins is not None:
+
+            zlabel = pytplot.data_quants[n].zaxis_opt['axis_label']
+            try:
+                ztype = pytplot.data_quants[n].zaxis_opt['z_axis_type']
+            except:
+                ztype = pytplot.default_values['z_axis_type']
+            try:
+                ytype = pytplot.data_quants[n].yaxis_opt['y_axis_type']
+            except:
+                ytype = pytplot.default_values['y_axis_type']
+            plot_labels[n] = [zlabel, ytype, ztype]
+    return plot_labels
+
+
+def get_bins(var):
+    # Get bins to be plotted.
+    bins = list()
+    for name, values in pytplot.data_quants[var].spec_bins.iteritems():
+        # name = variable name
+        # value = data in variable name
+        bins.append(values.values[0])
+    return bins
+
+
+def get_z_t_values(var):
+    # Get data to be plotted and time data for indexing.
+    time_values = list()
+    z_values = list()
+    for r, rows in pytplot.data_quants[var].data.iterrows():
+        # r = time
+        # rows = the flux at each time, where each row signifies a different time
+        time_values.append(r)
+        z_values.append(rows.values)
+    return time_values, z_values
+
+
+def Interactive2DPlot():
```

Comment:
Usually function names are are not capitalized, that is reserved for classes:

https://www.python.org/dev/peps/pep-0008/#class-names

I know it sounds nit picky, but this convention is actually used everywhere.

Reply:
Nah, that's fair, I'll be sure to change it.